src/plugins/guided_onboarding/public/components/guide_panel_step.tsx

```diff
@@ -42,7 +42,8 @@ const renderDescription = (description: string | StepDescriptionWithLink) => {
   const { descriptionText, linkText, linkUrl, isLinkExternal } = description;
   return (
     <>
-      {descriptionText}{' '}
+      {descriptionText}
+      <br />
```

Comment:
I'm changing a space to a line break here to fix an issue when the link is broken in weird places, see screenshot attached.
 
<img width="458" alt="Screenshot 2023-02-06 at 12 22 08" src="https://user-images.githubusercontent.com/6585477/216959302-deb53863-7372-41ad-8da3-ba51bd452a99.png">

Reply:
Why not `<EuiSpacer size="s" />`?

Reply:
I thought that the link should go on a new line, not separated from the text too much, so I think for that the "line break" element makes most sense.